package.json

```diff
@@ -51,21 +51,23 @@
     "eslint-config-prettier": "^6.5.0",
     "eslint-config-ts-react-important-stuff": "^1.1.0",
     "eslint-plugin-prettier": "^3.1.1",
+    "eslint-plugin-react-hooks": "^2.2.0",
     "fork-ts-checker-webpack-plugin": "^2.0.0",
     "husky": "^3.0.9",
     "jest": "^24.9.0",
     "jest-cli": "^24.9.0",
     "prettier": "^1.18.2",
     "pretty-quick": "^2.0.0",
     "ramda": "^0.26.1",
+    "react-dom": "^16.11.0",
```

Comment:
This should be a `dependency`, not a `devDependency`. See https://github.com/openmrs/packmap#explanation-of-packmap